src/main/java/org/radarbase/management/service/UserService.kt

```diff
@@ -166,20 +154,9 @@ class UserService @Autowired constructor(
         user.password = passwordService.generateEncodedPassword()
         user.resetKey = passwordService.generateResetKey()
         user.resetDate = ZonedDateTime.now()
-        user.activated = true
+        user.activated = false
```

Comment:
When do we activate users?

Reply:
We activate them after they verify their email.